py/picca/delta_extraction/expected_fluxes/dr16_expected_flux.py

```diff
@@ -471,17 +484,32 @@ def compute_delta_stack(self, forests, stack_from_deltas=False):
         w = stack_weight > 0
         stack_delta[w] /= stack_weight[w]
 
-        self.get_stack_delta = interp1d(Forest.log_lambda_grid[stack_weight > 0.],
-                                        stack_delta[stack_weight > 0.],
-                                        kind="nearest",
-                                        fill_value="extrapolate")
+        self.get_stack_delta = interp1d(
+            Forest.log_lambda_grid[stack_weight > 0.],
+            stack_delta[stack_weight > 0.],
+            kind="nearest",
+            fill_value="extrapolate")
         self.get_stack_delta_weights = interp1d(
             Forest.log_lambda_grid[stack_weight > 0.],
             stack_weight[stack_weight > 0.],
             kind="nearest",
             fill_value=0.0,
             bounds_error=False)
 
+    # TODO: We should check if we can directly compute the mean continuum
```

Comment:
this would mean getting mean_cont instead of iterative refinement of previous mean cont estimate?